ta/pkcs11/include/pkcs11_ta.h

```diff
@@ -469,6 +469,42 @@ struct pkcs11_mechanism_info {
 #define PKCS11_CKFM_EC_UNCOMPRESS		(1U << 24)
 #define PKCS11_CKFM_EC_COMPRESS			(1U << 25)
 
+/*
+ * pkcs11_object_head - Header of object whose data are serialized in memory
+ *
+ * An object is made of several attributes. Attributes are stored one next to
+ * the other with byte alignment as a serialized byte array. The byte array
+ * of serialized attributes is prepended with the size of the attrs[] array
+ * in bytes and the number of attributes in the array, yielding the struct
+ * pkcs11_object_head. Appended attributes byte arrays are wrapped with
+ * this header structure that defines the number of attribute items and the
+ * overall byte size of byte array field pkcs11_object_head::attrs.
```

Comment:
These 2 sentences seems redundant. Maybe remove one of them.

Reply:
OK, I'll remove the last